.github/workflows/gallery.yml

```diff
@@ -8,21 +8,33 @@ on:
       - master
 permissions:
   contents: write
-
+  id-token: write
+  pages: write
```

Comment:
Fix the error `Failed to create deployment (status: 403) with build version e2a388f6d28a33d6a27f024e26ead34696f32181. Ensure GITHUB_TOKEN has permission "pages: write".`